packages/http-client/src/remote-admin-api.ts

```diff
@@ -7,40 +7,57 @@ import { StreamID } from '@ceramicnetwork/streamid'
 export class RemoteAdminApi implements AdminApi {
   // Stored as a member to make it easier to inject a mock in unit tests
   private readonly _fetchJson: typeof fetchJson = fetchJson
+
   readonly baseURL = './admin/models'
 
   constructor(
-    private readonly _apiUrl: URL
+    private readonly _apiUrl: URL,
+    private readonly _context: Context
   ) {}
 
   private getUrl(): URL {
     return new URL(this.baseURL, this._apiUrl)
   }
 
+  private async buildAuthorizationHeader(modelsIDs?: Array<StreamID>): Promise<string> {
+    const body = modelsIDs ? modelsIDs.map(streamID => streamID.toString()) : undefined
+    const jws = await this._context.did.createJWS({
```

Comment:
Depending on what DID Provider is being used, this might result in a signature request popup right?  If so, we should make sure the contents of that signature request are at least vaguely human readable